manifests/cli/plugin.pp

```diff
@@ -9,7 +9,7 @@
 # $version:: The package version to ensure
 #
 define foreman::cli::plugin (
-  String $package = "${foreman::cli::hammer_plugin_prefix}${title}",
+  String $package = "${foreman::cli::params::hammer_plugin_prefix}${title}",
```

Comment:
Technically this doesn't have to change due to inheritance.

Reply:
Maybe. I was blindly copying from https://github.com/theforeman/puppet-foreman_proxy/commit/935d45ab53e59fa5424a60eb54b29ccf6b666313 ;)

Reply:
Honestly, if I'd read `foreman::cli::hammer_plugin_prefix` here, and wouldn't find it in `cli.pp`, I'd be confused at first, so I think explicitly using the one from `params` seems friendlier to the reader to me

Reply:
I've seen others confused by that as well so I guess it's ok.